cocos/particle/index.ts

```diff
@@ -9,13 +9,21 @@ import { ParticleSystem } from './particle-system';
 import { ParticleUtils } from './particle-utils';
 import CurveRange from './animator/curve-range';
 import { legacyCC } from '../core/global-exports';
+import { ParticleSystem2D } from './2d/particle-system-2d';
+import { MotionStreak } from './2d/motion-streak-2d';
+import { MotionStreakAssemblerManager } from './2d/motion-streak-2d-assembler';
+import { ParticleSystem2DAssembler } from './2d/particle-system-2d-assembler';
```

Comment:
Please separate to `core/particle-2d` module

Reply:
> Please separate to `core/particle-2d` module

Fixed.